### ISSUE TITLE
1.8.2 ps4 binary error

### ISSUE THREAD

**Issue by antonalog** [ispc/ispc#1130]
Hi there, I get an odd error with the PS4 version of 1.8.2 that I didn't get with 1.8.1, can't seem to compile anything.  

ispc.exe collide.ispc --target=sse4-i32x4 -g -o .\collide.o
CUSTOMBUILD : error : The requested CPU is incompatible with the CPU sse4-i32x4 needs: btver2 vs. corei7!

Any ideas?

**Comment by dbabokin** [comment]
Yes, it was reported and it's fixed. Let me build updated version.

**Comment by antonalog** [comment]
yes please! thanks!

**Comment by dbabokin** [comment]
I'll build based on current ISPC trunk + LLVM trunk. Does this sound ok for
you?

Another option is ISPC 1.8.2 + LLVM 3.7.

**Comment by antonalog** [comment]
I think our version of the PS4 SDK is currently LLVM 3.6.1 ?  Is it compatible with 3.7?  Thanks!

**Comment by dbabokin** [comment]
I'm not sure that it's connected somehow. Whether 3.7 or trunk (3.8), it should be compatible. I don't see how it can be dependent. Though I've never built and run PS4 programs, I don't have required setup available to me. I'll build 1.8.2+LLVM3.7 then.

**Comment by antonalog** [comment]
Okay that sounds great, I'll give it a try. Sorry I'm not really an LLVM expert, was very pleased when the last version just magically worked out of the box. :)

**Comment by dbabokin** [comment]
Here's the binary: https://drive.google.com/file/d/0Bxh4sVF04yhxS0w1azdVNy1ha1k/view?usp=sharing

Could you try it out and let me know how it works for you?

**Comment by antonalog** [comment]
Yes that's working again now, thank you!

**Comment by dbabokin** [comment]
This is ISPC trunk + LLVM 3.7. Please let me know about any issues with this distribution. That's important, as I don't have a chance to test PS4 build on real hardware.

**Comment by antonalog** [comment]
Okay sure, I'll let you know if I find anything, my little test case is currently running on an actual PS4, so it does work. :)

**Comment by zephyr111** [comment]
I am closing this issue since it seems done and there is no similar reported issue since 10 years. Feel free to reopen the issue or open a new one if needed.
